modules/govuk_elasticsearch/spec/classes/govuk_elasticsearch_housekeeping_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rspec'
+require_relative '../../../../spec_helper'
+require 'rspec-puppet'
+
+  describe 'govuk_elasticsearch::housekeeping', :type => :class do
+
+    let(:params) {{
+        :enabled  => true,
+        :es_repos => [
+            'kibana-lint',
+            'kibana-mint',
+            'kibana_tint'
+        ]
+    }}
+
+    context 'Checking what repositories variable expands like' do
+
+      it { is_expected.to contain_file('es-prune-snapshots').with_path('/usr/local/bin/es-prune-snapshots').with_content(/("kibana-lint" "kibana-mint" "kibana_tint")/) }
```

Comment:
Made the changes.  I had to keep the `/ /` because rspec doesn't respect word boundaries.